### PR TITLE
Event entry for nf-4bioimage workhop

### PR DESCRIPTION
**Pull request by mbruhns**
Hi,

I added an entry for the upcoming [bioimaging workshop](https://docs.google.com/forms/d/e/1FAIpQLSeGISw62vXgaM9DWPt-w6jlPIpuoROUMVGItNssuQVQ25AYtQ/viewform). Let me know if any changes are needed.

Best,
Matthias

### CODE REVIEW

File: sites/main-site/src/content/events/2024/bioimage_workshop.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+---
+title: Bioimage Analysis Workshop - September 2024 (Tübingen)
+subtitle: An in-person and online bioimage analysis workshop in Tübingen
+type: workshop
```
Comment (ewels) [inline sites/main-site/src/content/events/2024/bioimage_workshop.md:4]: Has to be one of `'bytesize' | 'talk' | 'hackathon' | 'training'`

```suggestion
type: hackathon
```

**Comment by ewels** [comment]
@nf-core-bot fix linting

File: sites/main-site/src/content/events/2024/bioimage_workshop.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- **MACSima/CODEX**
+- **MRI/CT scale imaging**
+
+The goal is to foster collaboration and exchange knowledge across the bioimage and Nextflow communities.
+
+## Schedule
+
+The event will be spread across two days, with a mix of presentations and practical sessions:
+
+<table>
```
Comment (ewels) [inline sites/main-site/src/content/events/2024/bioimage_workshop.md:37]:
```suggestion
<table class="table">
```

**Comment by ewels** [comment]
Sorry @mbruhns, lost track of this one 😢 

PR fixed up and working, will get it live before the event at least! 😅

**Comment by luiskuhn** [comment]
@ewels thanks a lot for all of your support! its highly appreciated! :)

**Pull request merged**